### PR TITLE
feat: bump plasma contract sha

### PR DESCRIPTION
**Pull request by nicholasmueller**
docker to use plasma contracts sha that allows setting minexitperiod env var

https://github.com/omisego/plasma-contracts/commit/a6d870f4562b359bece7bea01643e4a6b567b5a9

### CODE REVIEW

**Comment by unnawut** [comment]
I'm taking over this PR to fix the failing builds

**Comment by unnawut** [comment]
Blocked by https://github.com/omisego/elixir-omg/pull/1100

File: mix.lock [hunk 1]
```diff
@@ -55,7 +55,7 @@
   "parse_trans": {:hex, :parse_trans, "3.3.0", "09765507a3c7590a784615cfd421d101aec25098d50b89d7aa1d66646bc571c1", [:rebar3], [], "hexpm"},
   "phoenix": {:hex, :phoenix, "1.4.7", "e5bc9adcfa4c75a60d0952817d015a6caecc2a8812f4c53c1b61776dfa91d8ce", [:mix], [{:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: true]}, {:phoenix_pubsub, "~> 1.1", [hex: :phoenix_pubsub, repo: "hexpm", optional: false]}, {:plug, "~> 1.8.1 or ~> 1.9", [hex: :plug, repo: "hexpm", optional: false]}, {:plug_cowboy, "~> 1.0 or ~> 2.0", [hex: :plug_cowboy, repo: "hexpm", optional: true]}, {:telemetry, "~> 0.4", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm"},
   "phoenix_pubsub": {:hex, :phoenix_pubsub, "1.1.2", "496c303bdf1b2e98a9d26e89af5bba3ab487ba3a3735f74bf1f4064d2a845a3e", [:mix], [], "hexpm"},
-  "plasma_contracts": {:git, "https://github.com/omisego/plasma-contracts", "ea36f5ff46ab72ec5c281fa0a3dffe3bcc83178b", [branch: "master", sparse: "plasma_framework/contracts/"]},
+  "plasma_contracts": {:git, "https://github.com/omisego/plasma-contracts", "a6d870f4562b359bece7bea01643e4a6b567b5a9", [branch: "master", sparse: "plasma_framework/contracts/"]},
```
Comment (InoMurko) [inline mix.lock]:
```suggestion
  "plasma_contracts": {:git, "https://github.com/omisego/plasma-contracts", "1608296ef56ab98d857321a687c10dcdd86d72d3", [branch: "master", sparse: "plasma_framework/contracts/"]},
```

latest master from plasma-contracts

File: plasma-contract.Dockerfile [hunk 2]
```diff
@@ -12,6 +12,6 @@ RUN apk add --update \
 		git
 
 RUN git clone https://github.com/omisego/plasma-contracts.git
-RUN cd /home/node/plasma-contracts && git reset --hard ea36f5ff46ab72ec5c281fa0a3dffe3bcc83178b
+RUN cd /home/node/plasma-contracts && git reset --hard a6d870f4562b359bece7bea01643e4a6b567b5a9
```
Comment (InoMurko) [inline plasma-contract.Dockerfile]:
```suggestion
RUN cd /home/node/plasma-contracts && git reset --hard 1608296ef56ab98d857321a687c10dcdd86d72d3
```

**Pull request merged**